client/src/components/ReportMap.js

```diff
@@ -2,15 +2,17 @@ import API from "api"
 import { gql } from "apollo-boost"
 import Leaflet from "components/Leaflet"
 import {
-  PageDispatchersPropType,
   mapPageDispatchersToProps,
+  PageDispatchersPropType,
   useBoilerplate
 } from "components/Page"
-import _escape from "lodash/escape"
 import { Location } from "models"
 import PropTypes from "prop-types"
 import React, { useEffect, useMemo } from "react"
 import { connect } from "react-redux"
+import ReactDOM from "react-dom"
+import Report from "models/Report"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).